specification/cognitiveservices/data-plane/Language/stable/2022-05-01/conversationalanalysisauthoring.json

```diff
@@ -0,0 +1,3060 @@
+          },
+          {
+            "$ref": "common.json#/parameters/SkipParameter"
+          },
+          {
+            "$ref": "common.json#/parameters/MaxPageSizeParameter"
+          },
+          {
+            "$ref": "common.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The list of supported prebuilt entities.",
+            "schema": {
+              "$ref": "#/definitions/ConversationalAnalysisAuthoringPrebuiltEntities"
+            }
+          },
+          "default": {
+            "description": "Error response.",
+            "schema": {
+              "$ref": "common.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Successful Get Supported Prebuilt Entities": {
+            "$ref": "./examples/conversationalanalysisauthoring/SuccessfulGetSupportedPrebuiltEntities.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink",
+          "itemName": "value"
+        }
+      }
+    },
+    "/authoring/analyze-conversations/projects/global/training-config-versions": {
+      "get": {
+        "description": "Lists the support training config version for a given project type.",
+        "operationId": "ConversationalAnalysisAuthoring_ListTrainingConfigVersions",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/ConversationalAnalysisAuthoringProjectKindInQuery"
+          },
+          {
+            "$ref": "common.json#/parameters/TopParameter"
+          },
+          {
+            "$ref": "common.json#/parameters/SkipParameter"
+          },
+          {
+            "$ref": "common.json#/parameters/MaxPageSizeParameter"
+          },
+          {
+            "$ref": "common.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "List of config versions.",
+            "schema": {
+              "$ref": "#/definitions/ConversationalAnalysisAuthoringTrainingConfigVersions"
+            }
+          },
+          "default": {
+            "description": "Error response.",
+            "schema": {
+              "$ref": "common.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Successful Get Supported Training Config Versions": {
+            "$ref": "./examples/conversationalanalysisauthoring/SuccessfulGetSupportedTrainingConfigVersions.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink",
+          "itemName": "value"
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ConversationalAnalysisAuthoringCompositionSetting": {
+      "enum": [
+        "returnLongestOverlap",
+        "requireExactOverlap",
+        "separateComponents",
+        "combineComponents"
```

Comment:
Nit: most of the enum values I'm seeing in other Language service swaggers are using PascalCase. Would be good to be consistent throughout Language.

Reply:
QnA went GA with all the enums being camelCase, except for the error codes and the string index type. Text Analytics has a mix between camelCase and PascalCase. If we want to have the consistency across the services, then TA should convert all their enums to be camelCase instead.